opentelemetry/proto/profiles/v1development/eventprofile.md

```diff
@@ -0,0 +1,284 @@
+# Proposal: Support Timestamped Profiling Events
+
+**Author(s):** @felixge in collaboration with @thegreystone, @jbachorik et al.
+
+**🚧 WORK IN PROGRESS 🚧:** This proposal is still in the early stages. I'm mostly looking for feedback from active participants in the profiling SIG before soliciting feedback from the wider community.
+
+## Summary
+
+The OpenTelemetry profiling format is currently designed to record aggregated summaries of events. This proposal suggests to pivot the format toward recording individual events with timestamps while retaining the ability to record aggregated profiling data as well.
+
+This will enable the following use cases.
+
+1. Richer analysis of On-CPU as well as Off-CPU data.
```

Comment:
> Richer analysis of aggregated data, like On-CPU profiling

Not sure I follow. The use case enabled by this proposal is event based CPU profiling rather than aggregation. We already have aggregation?

Reply:
OTel ebpf profiler could aggregate on CPU profiling data. This is possible by setting [Sample.value](https://github.com/open-telemetry/opentelemetry-proto/blob/793e43e25f2eb8063c2693bc2e66e16672cf7ed8/opentelemetry/proto/profiles/v1development/profiles.proto#L379-L385) to a different value than 1 and provide a timestamp in [Sample.timestamps_unix_nano](https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/profiles/v1development/profiles.proto#L394C19-L394C39) for each `Sample.value`. 
Currently, this aggregation is not done as it was rejected during the review process and so it is still a low hanging fruit for optimization.